index.bs

```diff
@@ -346,8 +349,10 @@ When this method is invoked, the user agent MUST run the following steps:
             1. Abort these steps.
         1. Let |session| be a new {{XRSession}} object.
         1. [=Initialize the session=] with |session|, |mode|, and |device|.
-        1. [=Resolve the requested features=] given by |options|' {{XRSessionInit/requiredFeatures}} and |options|' {{XRSessionInit/optionalFeatures}} values for |session|, and let |resolved| be the returned value.
-        1. If |resolved| is <code>false</code>, run the following steps:
+        1. Let |descriptor| be an {{XRPermissionDescriptor}} initialized with |session|, |options|' {{XRSessionInit/requiredFeatures}}, and |options|' {{XRSessionInit/optionalFeatures}}.
+        1. Let |status| be an {{XRPermissionStatus}}, initially <code>null</code>
+        1. [=Request the xr permission=] with |descriptor| and |status|.
```

Comment:
It seems that "Request the xr permission" could return "status" instead of having it as in in/out parameter.

Reply:
It can't, that's how the permission request algorithm works. I agree that it is awkward, but that's a problem for upstream.

https://wicg.github.io/permissions-request/#permission-request-algorithm

Reply:
This is a different spec FWIW. You could re-use https://wicg.github.io/permissions-request/#boolean-permission-request-algorithm the same way https://w3c.github.io/permissions/#media-devices does.

Reply:
The media spec is using it incorrectly: it operates on the return value of the boolean permissions request algorithm, which does not exist, and it returns a value from its own permissions request algorithm, which according to the permissions spec should not exist. I'd be more than happy to clean this up but as written the permissions spec forces you to use an inout parameter. Perhaps it should be changed to use return values in many places so that it can be used in the way the media spec expects it to be.

![image](https://user-images.githubusercontent.com/1617736/75225879-412a7580-5760-11ea-8bec-e601145004e4.png)